dotnet/src/SemanticKernel/Orchestration/CancellationTokenExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel.Orchestration;
+
+#pragma warning disable IDE0130
+// ReSharper disable once CheckNamespace - Using the namespace of CancellationToken
+namespace System.Threading;
+#pragma warning restore IDE0130
+
+public static class CancellationTokenExtensions
```

Comment:
If we want to use these helpers internally in SK, that's fine and we can make these internal.  But I don't think SK should be adding such non-SK-related helpers to its public surface area.

Reply:
@dluc if these are handy helpers that we want to be "internal" but available throughout our projects, you can drop them in this folder where they'll be included as source (not as project references): dotnet\src\InternalUtilities